activities/tool-management/mailing-lists.md

```diff
@@ -0,0 +1,60 @@
+---
+type: resource, tooling
+---
+
+# Mailman mailing lists
+
+Mailman is free software for managing electronic mail discussion and e-newsletter lists. We use Mailman because it supports built-in archiving, automatic bounce processing, content filtering, digest delivery, spam filters, and because it's open-source.
```

Comment:
```suggestion
Mailman is free software for managing electronic mail discussion and e-newsletter lists. We use Mailman because it supports built-in archiving, automatic bounce processing, content filtering, digest delivery and spam filters, and because it's open-source.
```